scripts/make-guac.sh

```diff
@@ -376,6 +384,24 @@ then
         echo "ldap-username-attribute: ${LDAP_USER_ATTRIBUTE}"
         echo "ldap-config-base-dn:     ${LDAP_CONFIG_BASE},${LDAP_DOMAIN_DN}"
     ) >> /etc/guacamole/guacamole.properties
+    
+    if [ -n "$LDAP_GROUP_BASE" ]
+    then
+        log "Adding the LDAP group base DN, RBAC is enabled."
+        (
+            echo "ldap-group-base-dn       ${LDAP_GROUP_BASE},${LDAP_DOMAIN_DN}"
+        ) >> /etc/guacamole/guacamole.properties
+        
+        if [ "$GUAC_VERSION" == "0.9.7" ]
+        then
+            log "Enabling custom RBAC jar for 0.9.7"
+            rm -rf "/etc/guacamole/extensions/*"
+            cd "/etc/guacamole/extensions/"
+            curl -s -O https://s3.amazonaws.com/dicelab-guacamole/guacamole-auth-ldap-0.9.7.jar
+        else
+            log "Warning: Unknown RBAC support in this GUAC version, not 0.9.7!"
```

Comment:
bash will throw up on exclamation points in double quotes. Need to use single quotes:

`log "Warning: Unknown RBAC support in this GUAC version, not 0.9.7"'!'`

But I have not yet tried passing an exclamation point through the `log` function, so probably want to test that.